### PR TITLE
Override time on Sparkle & venues visited

### PR DESCRIPTION
**Pull request by whoaitsaimz**
Specific request for GitHub

### CODE REVIEW

**Comment by whoaitsaimz** [comment]
This is a test of our emergency CSS System (lol). @jonboutelle can u push button plz

**Review by jonboutelle — APPROVED** [review]
Approved

**Comment by 0xdevalias** [comment]
@whoaitsaimz @jonboutelle this PR should never have been merged. This is functionality that is configurable in the admin panel using the ‘show badges’ toggle. Please revert it.

File: src/scss/customOverrides.scss [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
 // This file exists for custom styles to deal with environment specific hacks/overrides
+
+.Badges {
+  &__visits {
+    display: none;
+  }
+  &__visit {
+    display: none;
+  }
+}
```
Comment (0xdevalias) [inline src/scss/customOverrides.scss:12]: @whoaitsaimz What are these styles even targeting? Searching the codebase for `__visit` or `__visits` doesn't even turn anything up for me..?

**Comment by 0xdevalias** [comment]
> This is functionality that is configurable in the admin panel using the ‘show badges’ toggle. Please revert it.

References:

- https://github.com/sparkletown/sparkle/search?q=%3CBadges
  - `src/components/organisms/UserProfileModal/UserProfileModal.tsx`
    - [Conditionally renders based on `venue?.showBadges`](https://github.com/sparkletown/sparkle/blob/0b519dddd0e5bcf3f3afd7107c6e4c55e72048c5/src/components/organisms/UserProfileModal/UserProfileModal.tsx#L148-L150)
  - `src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx`
    - [Conditionally renders based on `venue?.showBadges`](https://github.com/sparkletown/sparkle/blob/0b519dddd0e5bcf3f3afd7107c6e4c55e72048c5/src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx#L199)

**Pull request merged**